.github/workflows/build-images.yml

```diff
@@ -1,10 +1,14 @@
 name: Build images
 
 on:
-  # does not work with forks since the jobs will try to push to the target
-  # registry but run only with the privileges of the fork.
+  # on pull_request does not work with forks since the jobs will try to push
+  # to the target registry but run only with the privileges of the fork.
   #- pull_request
-  - push
+  push:
+    branches:
+      - '**'
```

Comment:
This means „on all branches“?

Reply:
Yes. If I leave this out it would **only** build on tags and not on branches anymore.

Reply:
Yeah right that’s what I thought.